### DISCUSSION TITLE
Will DeePKS launch a pre-training model?

### DISCUSSION THREAD

**Discussion by scott-5** [deepmodeling/abacus-develop#1766] [Ideas]
As we known, DPA-1(or GPT) is a pre-training model for reducing overhead of training set. So I have an idea that if there is a pre-training model for DeePKS, we can get higher accuracy (e.g. hybrid functionals) but lower calculating costs without training step or just need a few training set?
